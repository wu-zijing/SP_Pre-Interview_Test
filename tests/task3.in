4
23
2
9753
2468