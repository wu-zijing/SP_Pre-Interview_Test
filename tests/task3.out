["ad", "ae", "af", "bd", "be", "bf", "cd", "ce", "cf"]
["a", "b", "c"]
["wpjd", "wpje", "wpjf", "wpkd", "wpke", "wpkf", "wpld", "wple", "wplf", "wqjd", "wqje", "wqjf", "wqkd", "wqke", "wqkf", "wqld", "wqle", "wqlf", "wrjd", "wrje", "wrjf", "wrkd", "wrke", "wrkf", "wrld", "wrle", "wrlf", "wsjd", "wsje", "wsjf", "wskd", "wske", "wskf", "wsld", "wsle", "wslf", "xpjd", "xpje", "xpjf", "xpkd", "xpke", "xpkf", "xpld", "xple", "xplf", "xqjd", "xqje", "xqjf", "xqkd", "xqke", "xqkf", "xqld", "xqle", "xqlf", "xrjd", "xrje", "xrjf", "xrkd", "xrke", "xrkf", "xrld", "xrle", "xrlf", "xsjd", "xsje", "xsjf", "xskd", "xske", "xskf", "xsld", "xsle", "xslf", "ypjd", "ypje", "ypjf", "ypkd", "ypke", "ypkf", "ypld", "yple", "yplf", "yqjd", "yqje", "yqjf", "yqkd", "yqke", "yqkf", "yqld", "yqle", "yqlf", "yrjd", "yrje", "yrjf", "yrkd", "yrke", "yrkf", "yrld", "yrle", "yrlf", "ysjd", "ysje", "ysjf", "yskd", "yske", "yskf", "ysld", "ysle", "yslf", "zpjd", "zpje", "zpjf", "zpkd", "zpke", "zpkf", "zpld", "zple", "zplf", "zqjd", "zqje", "zqjf", "zqkd", "zqke", "zqkf", "zqld", "zqle", "zqlf", "zrjd", "zrje", "zrjf", "zrkd", "zrke", "zrkf", "zrld", "zrle", "zrlf", "zsjd", "zsje", "zsjf", "zskd", "zske", "zskf", "zsld", "zsle", "zslf"]
["agmt", "agmu", "agmv", "agnt", "agnu", "agnv", "agot", "agou", "agov", "ahmt", "ahmu", "ahmv", "ahnt", "ahnu", "ahnv", "ahot", "ahou", "ahov", "aimt", "aimu", "aimv", "aint", "ainu", "ainv", "aiot", "aiou", "aiov", "bgmt", "bgmu", "bgmv", "bgnt", "bgnu", "bgnv", "bgot", "bgou", "bgov", "bhmt", "bhmu", "bhmv", "bhnt", "bhnu", "bhnv", "bhot", "bhou", "bhov", "bimt", "bimu", "bimv", "bint", "binu", "binv", "biot", "biou", "biov", "cgmt", "cgmu", "cgmv", "cgnt", "cgnu", "cgnv", "cgot", "cgou", "cgov", "chmt", "chmu", "chmv", "chnt", "chnu", "chnv", "chot", "chou", "chov", "cimt", "cimu", "cimv", "cint", "cinu", "cinv", "ciot", "ciou", "ciov"]